test/integration/controller/jobs/pod/pod_controller_test.go

```diff
@@ -655,6 +655,7 @@ var _ = ginkgo.Describe("Pod controller", ginkgo.Ordered, ginkgo.ContinueOnFailu
 
 				ginkgo.By("set the pods as running", func() {
 					util.SetPodsPhase(ctx, k8sClient, corev1.PodRunning, pod1, pod2)
+					util.SetNodeName(ctx, k8sClient, "node1", pod1, pod2)
```

Comment:
```suggestion
					util.AssignPodToNode(ctx, k8sClient, "node1", pod1, pod2)
```

Also, just to match the sequence of changes in a production environment, first assign to the node and then set the phase.

Reply:
yes, that works too